package.json

```diff
@@ -7,8 +7,8 @@
   ],
   "source": "ts/index.ts",
```

Comment:
Can / should `source` be removed?

Reply:
`source` is used as the [entry point for Microbundle](https://github.com/developit/microbundle#set-up-your-packagejson)